haiku-apps/projectconceptor/projectconceptor-0.1.0.recipe

```diff
@@ -40,7 +39,8 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	make -C src OBJ_DIR=objects $jobArgs
+	make app  -C src OBJ_DIR=objects $jobArgs
+	make plugins -C src OBJ_DIR=objects $jobArgs
```

Comment:
You might wish to combine these two lines:
```diff
-	make app  -C src OBJ_DIR=objects $jobArgs
-	make plugins -C src OBJ_DIR=objects $jobArgs
+	make -C src OBJ_DIR=objects app plugins $jobArgs
```
(I've built with both variants and they produce identical packages.)

Reply:
ok thanks.. because the other way around (withou first building app target) somehow most of the plugins didnt compiled. (i eg was missing Grapheditor... which is the main editor)